packages/android-lib/src/main/kotlin/com/outsystems/plugins/osgeolocation/model/OSGLOCLocationOptions.kt

```diff
@@ -0,0 +1,11 @@
+package com.outsystems.plugins.osgeolocation.model
+
+/**
+ * Data class representing the options passed to getCurrentPosition and watchPosition
+ */
+data class OSGLOCLocationOptions(
+    val timeout: Long = 5000,
+    val maximumAge: Long = 3000,
+    val enableHighAccuracy: Boolean = true,
```

Comment:
Should these values be set here? Aren't they OS-specific?

Reply:
I believe they can be set. They should come defined from the OutSystems (Cordova) and Capacitor bridges, but if they're not, then this way we make sure the library has default values set.

Reply:
The issue is that these values should be defined by the callers, not the library. Given the wide range of scenarios where this library might be used, location accuracy and response times are highly context-specific. Providing default values through an empty constructor could result in unpredictable behaviour and suboptimal outcomes in terms of precision, battery usage and CPU performance.
Additionally, relying on OS-provided defaults makes it look weird, especially for `enableHighAccuracy`, where both bridges use the setting oppositely.

Reply:
That makes sense. I removed the default values, can you check?